features/poller/monitors/core/src/main/java/org/opennms/netmgt/poller/monitors/SSLCertMonitor.java

```diff
@@ -222,27 +226,46 @@ public PollStatus poll(final MonitoredService svc, final Map<String, Object> par
                 for (int i = 0; i < certs.length && !serviceStatus.isAvailable(); i++) {
                     if (certs[i] instanceof X509Certificate) {
                         X509Certificate certx = (X509Certificate) certs[i];
+                        String subject = certx.getSubjectDN().getName();
```

Comment:
Are all of these guaranteed to be not null ?

Reply:
The javadocs do not explicitly say whether or not they can be null, so I might as well add a check.